doc/content/concepts/email-templates.md

```diff
@@ -2,7 +2,6 @@
 title: "Email Templates"
 description: ""
 weight: 15
-summary: Email templates define the contents of the emails that The Things Stack sends to its users. They allow network operators to override the default contents of the emails with their own custom contents.
```

Comment:
This is not redundant. Can you replace `The Things Stack` by the shortcode here?

Reply:
No, you can't. I think it would be better to remove the `What is it` header and add a `<!--more-->` below the first paragraph.

Reply:
I'm also fine with that. As long as it renders nicely on a tile